backend/src/services/github/github.service.ts

```
@@ -0,0 +1,59 @@
+    return GitHubService.instance;
+  }
+
+  async getRepositoryMetadata(): Promise<GitHubRepositoryMetadata> {
+    if (this.cachedMetadata && Date.now() < this.cacheExpiresAt) {
+      return this.cachedMetadata;
+    }
+
+    try {
+      const response = await axios.get(
+        'https://api.github.com/repos/InsForge/InsForge',
+        {
```

ESLint

[error] Replace `⏎········'https://api.github.com/repos/InsForge/InsForge',⏎·······` with `'https://api.github.com/repos/InsForge/InsForge',`

(prettier/prettier)